### PR TITLE
Introducing Blacklight Crawler

### PR DESCRIPTION
**Pull request by mevey**

### CODE REVIEW

File: aleph/crawlers/blacklight.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+
+class BlacklightCrawler(Crawler):
+
+    def crawl_document(self, base_url, id):
+        source_url = base_url + 'catalog/'+ id + '.json'
+        response = requests.get(source_url)
+        if response.status_code >= 500:
+            log.warning('Failed to load: %r', source_url)
+            time.sleep(10)
```
Comment (andkamau) [inline aleph/crawlers/blacklight.py]: Any reason for this, @mevey ?

File: aleph/crawlers/blacklight.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        for base_url in SITES:
+            print 'Working on base_url: {}'.format(base_url)
+            self.attributes = SITES[base_url]
+            self.label= self.attributes['label']
+            self.source = self.create_source(label = self.label)
+            self.failed_articles = 0
+            page_count = self.get_page_count(base_url)
+            print "Pages: {}".format(page_count)
+            page_number = 1
+            while (page_number <= page_count) and (page_number <= 2):
```
Comment (andkamau) [inline aleph/crawlers/blacklight.py]: @mevey  What's the rationale behind condition `page_number <= 2` ?

Comment (mevey) [inline aleph/crawlers/blacklight.py]: This was condition I put in place when testing. The goal was that I did not have to run through all possible pages. Just two.

I will remove this in my next commit.

File: aleph/crawlers/blacklight.py (hunk 1 above)
Comment (mevey) [inline aleph/crawlers/blacklight.py]: Not sure why I added it. But it'll be removed on next commit.

**Pull request merged**